LoanAppJPA/src/main/java/az/growlab/LoanApp/model/Client.java

```diff
@@ -0,0 +1,34 @@
+package az.growlab.LoanApp.model;
+
+import az.growlab.LoanApp.enums.ActionStatus;
+import az.growlab.LoanApp.enums.FinalStatus;
+import lombok.Data;
+
+import javax.persistence.*;
+
+@Data
+@Entity
+@Table(name = "client")
+public class Client {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @OneToOne
+    @JoinColumn(name = "passport_id")
+    private PassportInformation passportInfo;
+
+    @OneToOne
+    @JoinColumn(name = "personal_id")
+    private PersonalInformation personalInfo;
+
+    @OneToOne
+    @JoinColumn(name = "loan_id")
+    private LoanInformation loanInfo;
+
+    private String actionStatus;
```

Comment:
use enums

Reply:
can you write in more detail? I didn't understand "move special domain"

Reply:
move it to Order domain and associate client domain with order domain